web-local/src/lib/components/column-profile/column-types/sparks/TimestampSpark.svelte

```diff
@@ -41,14 +41,14 @@
   export let top = 12;
   export let bottom = 4;
 
-  export let buffer = 4;
-  export let leftBuffer = buffer;
-  export let rightBuffer = buffer;
-  export let topBuffer = buffer;
-  export let bottomBuffer = buffer;
+  // export let buffer = 4;
+  // export let leftBuffer = buffer;
+  // export let rightBuffer = buffer;
+  // export let topBuffer = buffer;
+  // export let bottomBuffer = buffer;
 
-  export let objectName: string;
-  export let columnName: string;
+  // export let objectName: string;
+  // export let columnName: string;
```

Comment:
What's with all these comments? If the code isn't needed, can we just delete?

Reply:
The component as a whole is not being actively used anywhere but we might be using these sparklines inside the dimension table at some point. I would prefer to have to them as is for now as we might be refactoring and using these components in the next quarter.

Reply:
Sure, sounds good.